src/Files.App/Views/MainPage.xaml

```diff
@@ -217,7 +217,13 @@
 
 			<!--  Content Header  -->
 			<sidebar:SidebarView.ContentHeader>
-				<Border>
+				<Border
```

Comment:
Changes look good but I think wrapping with Border increases amount of xaml tree.

Reply:
The border was already there before (it seems to be a requirement).